docs/docs/metrics.md

```diff
@@ -138,6 +138,9 @@ Dashes in metric names are replaced with underscores.
   received since the current Marathon instance became a leader.
 * `marathon.mesos.offers.used.counter` — the count of offers used since
   the current Marathon instance became a leader.
+* `marathon.mesos.task-states.<task_state>.counter` — the count of task
```

Comment:
what do you think about `tasks.state` ?

btw. now looking at the other metrics I see we are not very consistent in this...

We have `marathon.mesos.offer-operations`
But also we have `marathon.mesos.offers.declined`

I would personally prefer
`marathon.mesos.offers.operation`
`marathon.mesos.offers.type`

But it's totally up for discussion :)

Reply:
Though this is a possibility, we agreed that if parts of an entity name, that consists of two or more words, should be separated with dashes, not dots. In the light of this, say, `marathon.mesos.offers.incoming.counter` and `marathon.mesos.offer-operations.reserve.counter` do not seem to be inconsistent w.r.t. naming.

As for `marathon.mesos.task-states.<task-state>.counter`, I have been thinking of — besides the aforementioned one —

- `marathon.mesos.task-states.<task-state>.received.counter`
- `marathon.mesos.tasks.states.<task-state>.counter`
- `marathon.mesos.tasks.states.<task-state>.received.counter`

Which one do you like better? Another option is to use `incoming` instead of `received`.

Reply:
I like `received`.

Or...

`marathon.mesos.task-updates.<task-state>.counter` ?